app/apollo/resolvers/channel.js

```diff
@@ -390,13 +375,13 @@ const channelResolvers = {
             subscriptionsRbacSync( [subscriptionObj], { resync: false }, context ).catch(function(){/*ignore*/});
           }
           catch( e ) {
-            logger.error(e, `${queryName} failed to create subscription '${subscriptionObj.name}' when serving ${req_id}.`);
-            // Cannot fail here, the Version has already been created.  Continue.
+            logger.error(e, `${queryName} error creating subscription '${subscriptionObj.name}' when serving ${req_id}.`);
+            // Cannot fail here, the Channel has already been created.  Continue.
```

Comment:
Do we want to log an error here if this is an expected, continuable "exception?"

Reply:
It's not exactly an 'expected' failure -- it would typically mean something like mongo accepted the save of one record, then failed to accept the second.  It's continuable in the sense that we can't back up and delete the first record (chances are mongo is malfunctioning and it would be impossible), but this should still be reported as an error rather than a warning.